Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 authors = ["Kisio Digital <guillaume.pinot@kisio.org>"]
```

Comment:
I think you can add you names as the package owner :stuck_out_tongue_winking_eye: